### PR TITLE
Let evm in

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
dvm-evm means darwinia-virtual--machine-ethereum-virtual-machine?

```
pallet-balances -> darwinia-balances
pallet-staking -> darwinia-staking
...
pallet-evm -> darwinia-evm
```

**Comment by AsceticBear** [comment]
`darwinia-evm` Seems better.

File: frame/dvm/evm/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+frame-support    = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+frame-system     = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+pallet-timestamp = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+sp-core          = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+sp-io            = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+sp-runtime       = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+sp-std           = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
+
+[dev-dependencies]
+pallet-balances  = { default-features = false, git = "https://github.com/darwinia-network/substrate.git", branch = "common-library" }
```
Comment (AurevoirXavier) [inline frame/dvm/evm/Cargo.toml]: Should test with darwinia-balances

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]

File: frame/dvm/evm/Cargo.toml (hunk 1 above)
Comment (AsceticBear) [reply frame/dvm/evm/Cargo.toml]: Done, please check it again.

File: frame/dvm/evm/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+	/// Account balance.
+	pub balance: U256,
+	/// Full account storage.
+	pub storage: std::collections::BTreeMap<H256, H256>,
+	/// Account code.
+	pub code: Vec<u8>,
+}
+
+decl_storage! {
+	trait Store for Module<T: Trait> as EVM {
```
Comment (HackFisher) [inline frame/dvm/evm/src/lib.rs:278]: Should consider using a different storage name to identify from `evm` pallet.

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: frame/dvm/evm/src/lib.rs (hunk 2 above)
Comment (AsceticBear) [reply frame/dvm/evm/src/lib.rs]: Done.

**Review by HackFisher — APPROVED** [review]

**Comment by HackFisher** [comment]
@AurevoirXavier Please help review this pull request.

File: bin/node/cli/src/chain_spec.rs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // --- std ---
 use std::collections::BTreeMap;
 // --- substrate ---
-use pallet_evm::GenesisAccount;
+use darwinia_evm::GenesisAccount;
```
Comment (AurevoirXavier) [inline bin/node/cli/src/chain_spec.rs:4]: Move to darwinia part

File: bin/node/runtime/pangolin/src/lib.rs [hunk 4]
```diff
@@ -215,6 +215,9 @@ pub use wasm::*;
 use codec::{Decode, Encode};
 use static_assertions::const_assert;
 // --- substrate ---
+use darwinia_evm::{
```
Comment (AurevoirXavier) [inline bin/node/runtime/pangolin/src/lib.rs:218]: Move to darwinia part

**Review by AurevoirXavier — CHANGES_REQUESTED** [review]
Looks good, but the import section

**Review by AurevoirXavier — APPROVED** [review]

**Pull request merged**